### ISSUE TITLE
Terraform detects rancher cluster update while config is not changed

### ISSUE THREAD

**Issue by avthart** [rancher/terraform-provider-rancher2-archive#52]
I have created a rancher cluster with the vpshere cloud provider enabled, see below:

```
resource "rancher2_cluster" "cluster" {
  name = "${var.cluster_name}"
  description = "${var.cluster_description}"
  kind = "rke"
  rke_config {
    network {
      plugin = "canal"
    }
    cloud_provider {
      vsphere_cloud_provider {
        global {
          insecure_flag = true
        }
        virtual_center = [
          {
            name = "${var.vcenter_server}",
            user = "${var.vsphere_storage_user}",
            password = "${var.vsphere_storage_password}",
            datacenters = "${var.vsphere_datacenter}",
          }
        ]
        workspace {
          server = "${var.vcenter_server}"
          folder = "${var.vsphere_folder}"
          default_datastore = "${var.vsphere_storage_default_datastore}"
          datacenter = "${var.vsphere_datacenter}"
        }
      }
    }
  }
}
```

When I run terraform plan, terraform detects a change while the config is the same.

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ rancher2_cluster.cluster
      rke_config.0.cloud_provider.0.vsphere_cloud_provider.0.disk.#:    "1" => "0"
      rke_config.0.cloud_provider.0.vsphere_cloud_provider.0.network.#: "1" => "0"


Plan: 0 to add, 1 to change, 0 to destroy.
```

**Comment by rawmind0** [comment]
Submitted PR #55 to fix it.

**Comment by rawmind0** [comment]
@avthart, Merged PR #55 to fix it. Please, reopen issue if needed.

**Comment by avthart** [comment]
Thanks! I will build and test the latest version on the customers test
environment.

Op di 12 mrt. 2019 20:22 schreef Raúl Sánchez <notifications@github.com>:
